src/wp-includes/blocks.php

```diff
@@ -757,6 +757,33 @@ function get_hooked_blocks() {
 	return $hooked_blocks;
 }
 
+function insert_hooked_blocks( $relative_position, &$anchor_block, $hooked_blocks, $context ) {
```

Comment:
Re-sharing my comment to consider making `&$anchor_block` the first param: https://github.com/WordPress/wordpress-develop/pull/5523/files#r1381418436.